server/routes/caseNotes/caseNotesController.test.ts

```diff
@@ -104,26 +115,109 @@ describe.each([
           })
       })
     })
+
+    it('should combine the same users into a single call', async () => {
+      const caseNote1 = caseNoteFactory.build({
+        sentBy: { userId: '1', username: 'username', authSource: 'delius' },
+      })
+      const caseNote2 = caseNoteFactory.build({
+        sentBy: { userId: '1', username: 'username', authSource: 'delius' },
+      })
+      const caseNotePage: Page<CaseNote> = pageFactory.pageContent([caseNote1, caseNote2]).build() as Page<CaseNote>
```

Comment:
It looks like we're double-casting here? Do we need the `: Page<CaseNote>` and `as Page<CaseNote>`?

Reply:
I've changed it to just `as Page<CageNote>`